index.markdown

```diff
@@ -1,60 +0,0 @@
----
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
-layout: splash
-header:
-  overlay_color: "#5e616c"
-excerpt: >
-  NMIND is a collaborative dedicated to accelerating scientific discovery in neuroimaging research that was formed in 2020 as a grassroots initiative - responding to the growing frustration about redundancies in effort, continuing questions about the reproducibility of code by independent teams, and challenges arising in the sharing of code between teams due to lack of consistent nomenclature and conventions.
-guiding_principals_header:
-  - excerpt: "## Guiding Principles
-
-The NMIND solution has three guiding principles: Alignment, Certification, & Engagement."
```

Comment:
In https://github.com/nmind/nmind.github.io/blob/4bdcb184616a909f36a4cade129d4e58c8998859/index.md#L16-L28